paddle/optimizer/Tensor.h

```diff
@@ -0,0 +1,25 @@
+#ifndef PADDLE_FAKE_TENSOR_H_
+#define PADDLE_FAKE_TENSOR_H_
+/**
+ * @brief fake tensor for testing 
```

Comment:
Maybe "Tensor used by optimizer" is more appropriate. Since this tensor is actually working, not something fake and does not work. If you agree, please change header guard (PADDLE_FAKE_TENSOR_H_) as well.